ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -389,6 +391,12 @@ - (void)testShouldHandleFileOpEvent_RootHandle {
             true, // isDirectory,
             &testRootHandle,
             &pid));
+
+    if (VirtualizationRoot_IsValidRootHandle(testRootHandle))
```

Comment:
Why do we need to check `VirtualizationRoot_IsValidRootHandle`?

Couldn't  we always know at this point whether `testRootHandle` will be valid?  Or is this in case something above in the test fails and we don't want to poison things for the rest of the test?

Reply:
> Or is this in case something above in the test fails and we don't want to poison things for the rest of the test?

Exactly this.